code/game/machinery/gym.dm

```diff
@@ -0,0 +1,151 @@
+/obj/machinery/gym
+	name = "Advanced Arcade Machine"
+	desc = "Links with your brain to reduce lag to minimum. Now, success really does depend only on your skill!"
```

Comment:
```suggestion
	desc = "Links with your brain to reduce lag to minimum. Now, success really does depend only on your skill!"
	description_info = "Exercise machines can be used to either temporarily increase your stats or permanently using a rest point."
```
I dunno how to describe it better but it needs some explanation in its description info